src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -1454,11 +1456,9 @@ bool NetworkOPsImp::beginConsensus (uint256 const& networkClosed)
         return false;
     }
 
-    assert (prevLedger->getHash () == closingInfo.parentHash);
+    assert (prevLedger->info().hash == closingInfo.parentHash);
     assert (closingInfo.parentHash ==
-            m_ledgerMaster.getClosedLedger ()->getHash ());
-
-    prevLedger->setImmutable (app_.config());
```

Comment:
I'm assuming we know that `setImmutable` has already been called for the `prevLedger` so we don't need to call it here. I cannot confirm this is the case. You're confident `setImmutable` is not needed here?